pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -223,6 +226,20 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 			}
 			return []record.Record{{Name: "config/proxy", Item: ProxyAnonymizer{config}}}, nil
 		},
+		func() ([]record.Record, []error) {
+			requests, err := i.certClient.CertificateSigningRequests().List(metav1.ListOptions{})
+			if errors.IsNotFound(err) {
+				return nil, nil
```

Comment:
I know this is used in previous gathering functions, but is the `nil` handled properly later?

Reply:
Fortunately returning nil in a slice will return empty slice and wont fail. It looks even empty Record wont fail as its just a structure. I have just verified both.